src/s2s/pipeline.cpp

```
@@ -0,0 +1,1749 @@
+        out += under_thousand_to_words(static_cast<int>(value));
+    }
+    return out;
+}
+
+std::string
+decimal_to_words(double value) {
+    std::ostringstream ss;
+    ss << value;
+    const std::string repr = ss.str();
+    const size_t dot = repr.find('.');
+    if (dot == std::string::npos || repr.find_first_of("eE") != std::string::npos)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)